LiveRamp ATS SDK - Simple Demo/ViewController.swift

```diff
@@ -54,17 +53,21 @@ class ViewController: UIViewController {
         UserDefaults.standard.set(ccpaString, forKey:"IABUSPrivacy_String");
     }
     
+    
     func initializeATSSDK() {
         
-        // Sample test file available here: https://gist.github.com/jasonc-lr/092f50c425b898fc51ae6b55b765af2a
-        guard let fallbackConfigPath = Bundle.main.path(forResource: "fallback_configuration", ofType: "json"),
-        let jsonData = try? String(contentsOfFile: fallbackConfigPath).data(using: .utf8),
-        let fallbackConfiguration = try? JSONDecoder().decode(LRConfiguration.self, from: jsonData) else { return}
+        // Example workflow for how you determine whether you invoke hasConsentForNoLegislation
+        let doNotRequireCCPACheckInUS = true;
+        let supportOtherGeos = true;                // For handling initialization in a country that isn't US or EU
         
-        // Provide appId and fallback configuration (fallback will be removed in a future version_
-        let lrAtsConfiguration = LRAtsConfiguration(appId: appId, fallbackConfiguration: fallbackConfiguration)
+        if (doNotRequireCCPACheckInUS || supportOtherGeos) {
+            LRAts.shared.hasConsentForNoLegislation = true;
+        }
+     
+        // Provide just the appId - optional arg for isTestMode (by default, it'll be false)
+        let lrAtsConfiguration = LRAtsConfiguration(appId: appId, isTestMode: false);
```

Comment:
Public Note: Make sure isTestMode is always set to `false` before releasing to prod
If this is set to `true` you will get no network calls and fake envelopes.